source/App/source/ExampleHost.FunctionApp.Tests/Integration/FeatureManagementTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Net;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.AppConfiguration;
+using Energinet.DataHub.Core.TestCommon;
+using ExampleHost.FunctionApp.Tests.Fixtures;
+using ExampleHost.FunctionApp01.Common;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ExampleHost.FunctionApp.Tests.Integration;
+
+/// <summary>
+/// Tests verifying the configuration and behaviour of Feature Management (feature flags).
+/// </summary>
+public class FeatureManagementTests
+{
+    private const string MessageRoute = "api/message";
+
+    /// <summary>
+    /// Tests demonstrating use of a feature flag.
+    /// See the function triggered to understand how to use feature manager to switch on
+    /// a feature flag named <see cref="FeatureFlags.Names.UseGetMessage"/>.
+    /// </summary>
+    [Collection(nameof(ExampleHostsCollectionFixture))]
+    public class GetMessage
+    {
+        public GetMessage(ExampleHostsFixture fixture, ITestOutputHelper testOutputHelper)
+        {
+            Fixture = fixture;
+            Fixture.SetTestOutputHelper(testOutputHelper);
+
+            Fixture.App01HostManager.ClearHostLog();
+        }
+
+        private ExampleHostsFixture Fixture { get; }
+
+        [Theory]
+        [InlineData("false", "Disabled")]
+        [InlineData("true", "Enabled")]
+        public async Task Given_DisabledValueIs_When_Requested_Then_ExpectedContentIsReturned(string disabledValue, string expectedContent)
```

Comment:
Is this the feature flag value? I don't really get the `disabledValue` name, since the test has cases with it being enabled or disabled.

```suggestion
        public async Task Given_FeatureFlagEnabled_When_Requested_Then_ExpectedContentIsReturned(string featureFlagEnabled, string expectedContent)
```

Reply:
This was also copied from the Feature Management example and demonstrates how a developer can use the built in "disable" flag in Functions to disable/enable an Azure Function.